proposals/description/cr_oracle_cleanup.ts

```diff
@@ -0,0 +1,152 @@
+      description: 'Revoke MINTER_ROLE from optimisticMinter (fip-96 deprecates OA timelock)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_CONTROLLER_ROLE'), addresses.raiPCVDripController],
+      description: 'Revoke PCV_CONTROLLER_ROLE from raiPCVDripController (RAI deposits are empty)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_CONTROLLER_ROLE'), addresses.lusdPCVDripController],
+      description: 'Revoke PCV_CONTROLLER_ROLE from lusdPCVDripController (BAMMDeposit is deprecated)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('ORACLE_ADMIN_ROLE'), addresses.collateralizationOracleGuardian],
+      description:
+        'Revoke ORACLE_ADMIN_ROLE from collateralizationOracleGuardian (no more guardian updates of cached CR Oracle)'
+    },
+    {
+      target: 'tribeReserveStabilizer',
+      values: '0',
+      method: 'setCollateralizationOracle(address)',
+      arguments: (addresses) => [addresses.collateralizationOracle],
+      description: 'Set CR Oracle in Tribe Reserve Stabilizer (backstop) to the uncached CR Oracle'
+    },
+    {
+      target: 'pcvEquityMinter',
+      values: '0',
+      method: 'setCollateralizationOracle(address)',
+      arguments: (addresses) => [addresses.collateralizationOracle],
+      description: 'Set CR Oracle in PCV Equity Minter (buybacks) to the uncached CR Oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.volt, addresses.voltOracle],
+      description: 'Replace VOLT oracle in CR oracle'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [addresses.uniswapPCVDeposit, addresses.ethPSM, '10000'],
+      description: 'Withdraw remaining ETH/FEI liquidity from Uniswap v2'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.uniswapPCVDeposit, addresses.voltOracle],
+      description: 'Replace VOLT oracle in CR oracle'
+    },
+    /*{
+      target: 'raiPriceBoundPSM',
+      values: '0',
+      method: 'unpause()',
+      arguments: (addresses) => [],
+      description: 'Unpause RAI PSM'
+    },*/
+    /*{
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [addresses.ethTokemakPCVDeposit, addresses.ethPSM, '10000'],
+      description: 'Withdraw all ETH from Tokemak'
+    },*/
```

Comment:
why not include these two?

Reply:
not sure if this will be executed before in another TC or guardian action

Reply:
added tokemak withdrawal, RAI PSM content is moved somewhere else (RAI to TC multisig, FEI to DAI PSM for burn)